PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -138,16 +138,30 @@ def nanoAOD_addBoostedTauIds(process, idsToRun=[]):
 
     return process
 
-def nanoAOD_addPNetToTaus(process, addPNetInfo=False, runPNetCHSAK4=False):
-    if addPNetInfo:
+def nanoAOD_addUTagToTaus(process, addUTagInfo=False, runUTagAK4=False,
+                          usePUPPIjets=False):
+    if addUTagInfo:
         originalTauName = process.finalTaus.src.value()
-        updatedTauName = originalTauName+'WithPNet'
-        jetCollection = "updatedJets"
-        process.load('RecoBTag.ONNXRuntime.pfParticleNetFromMiniAODAK4_cff')
-        pnetTagName = "pfParticleNetFromMiniAODAK4CHSCentralJetTag"
-        pnetDiscriminators = [];
-        for tag in getattr(process,pnetTagName+"s").flav_names.value():
-            pnetDiscriminators.append(pnetTagName+"s:"+tag)
+        
+        if usePUPPIjets: # option to use PUPPI jets   
+            jetCollection = "updatedJetsPuppi"
+            TagName = "pfUnifiedParticleTransformerAK4JetTag"
```

Comment:
It seems that the code always uses `TagName+"s"` -- then why not just changing `TagName` itself to have the `s`?

Reply:
This is how things had been setup for the previous PR that added PNet scores - but indeed, I don't see a reason not to add the `s`